project/admin_dashboard/04_time_to_complete_loc.json

```diff
@@ -6,19 +6,33 @@
     "data": {
         "url": "dataset:userstats"
     },
-    "mark": "point",
     "transform": [
         {"filter": "datum.letter_submission_date !== null"},
         {
             "calculate": "round((time(datum.letter_submission_date) - time(datum.onboarding_date)) / (1000 * 60))",
             "as": "minutes"
-        }
+        },
+        {"filter": {"field": "minutes", "valid": true}}
```

Comment:
I don't think this line is actually needed, it's actually just left-over from a thing I was trying to debug.